src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -235,16 +235,7 @@ public static void Title_Get_Windows_NoNulls()
     {
         string title = Console.Title;
         string trimmedTitle = title.TrimEnd('\0');
-
-        if (PlatformDetection.IsWindowsNanoServer)
-        {
-            // Nano server titles are currently broken
-            Assert.NotEqual(trimmedTitle, title);
-        }
-        else
-        {
-            Assert.Equal(trimmedTitle, title);
-        }
+        Assert.Contains(title, trimmedTitle);
```

Comment:
Can this be Equal rather than Contains?  If not, does that mean there's still a difference on Nano?